src/us/kbase/workspace/kbase/ArgUtils.java

```diff
@@ -98,44 +96,52 @@ public static Instant chooseInstant(
 		if (epochInMilliSec != null) {
 			return Instant.ofEpochMilli(epochInMilliSec);
 		}
-		return null; // TODO CODE at some point switch to optionals
```

Comment:
Everything left that calls this method expects nulls as input so seems a bit pointless